### PR TITLE
1928 add a version reference in this repository

### PR DESCRIPTION
**Pull request by davidcr01**
|Related issue|
|---|
|https://github.com/wazuh/wazuh-packages/issues/1928|

## Description

Update README.md of wazuh-packages adding the "Compatibility matrix" section.
https://github.com/wazuh/wazuh-packages/tree/1928-add-a-version-reference-in-this-repository

## Logs example

image

### CODE REVIEW

**Review by verdx — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -27,6 +27,24 @@ In this repository, you can find the necessary tools to build a Wazuh package fo
 * `master` branch contains the latest code, be aware of possible bugs on this branch.
 * `stable` branch on correspond to the last Wazuh stable version.
 
+## Compatibility matrix
```
Comment (alberpilot) [inline README.md:30]:
```suggestion
## Distribution version matrix
```

File: README.md [hunk 2]
```diff
@@ -27,6 +27,24 @@ In this repository, you can find the necessary tools to build a Wazuh package fo
...
+| Wazuh dashboard | Opensearch dasbhoards |
+|-----------------|-----------------------|
+| 4.3.x           | 1.2.0                 |
+| 4.4.x           | 2.3.0                 |
+
+### Indexer
+
+| Wazuh indexer   | Opensearch            |
+|-----------------|-----------------------|
+| 4.3.x           | 1.2.0                 |
```
Comment (alberpilot) [inline README.md:45]:
```suggestion
| 4.3.x           | 1.2.4                 |
```
The indexer has a different patch in 4.3.x.

File: README.md [hunk 3]
```diff
@@ -27,6 +27,24 @@ In this repository, you can find the necessary tools to build a Wazuh package fo
 * `master` branch contains the latest code, be aware of possible bugs on this branch.
 * `stable` branch on correspond to the last Wazuh stable version.
 
+## Compatibility matrix
+
+The following table shows the references for the versions of each component.
+
+### Dashboard
+
+| Wazuh dashboard | Opensearch dasbhoards |
```
Comment (alberpilot) [inline README.md:36]:
```suggestion
| Wazuh dashboard | Opensearch dashboards |
```

**Review by alberpilot — CHANGES_REQUESTED** [review]
Please review the requested changes

**Review by verdx — APPROVED** [review]
LGTM, but where is this information taken from?

**Review by alberpilot — APPROVED** [review]

**Pull request merged**